src/foam/u2/borders/LoadingBorder.js

```diff
@@ -26,27 +47,53 @@ foam.CLASS({
       /* extra width covers right-side padding of wizard */
       width: calc(100% + 2*48px);
       height: 100%;
-      background-color: rgba(0,0,0,0.7 );
+      background-color: hsla(216,33%,97%,0.9);
 
       /* negative margin covers left-side padding of wizard */
       margin: 0 -48px;
       z-index: 1000;
+
+      /* ease-out animation makes things feel stable */
+      transition: all 200ms ease-out;
+
+      overflow: hidden;
     }
     ^overlay .foam-u2-LoadingSpinner {
       width: 50%;
       height: 50%;
+
+      /* ease-out animation makes things feel stable */
+      transition: all 200ms ease-out;
     }
     ^overlay .foam-u2-LoadingSpinner img {
       width: 100%;
       height: 100%;
     }
+
+    ^overlay^idle {
+      opacity: 0;
+      pointer-events: none;
+    }
+    ^overlay^pending {
+      opacity: 0.5;
+      pointer-events: none;
+    }
+    ^overlay^pending .foam-u2-LoadingSpinner {
+      width: 20%;
+      height: 20%;
+    }
   `,
 
   properties: [
+    // {
```

Comment:
don't see reason to keep this. If not needed would recommend removing commented out property